.github/workflows/test.yml

```
@@ -0,0 +1,350 @@
+          EOF
+      
+      - name: Run migrations
+        run: |
+          python manage.py migrate --noinput
+      
+      - name: Run ${{ matrix.test-suite.name }}
+        run: ${{ matrix.test-suite.command }}
+      
+      - name: Upload coverage to Codecov
+        if: matrix.test-suite.coverage == true
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)